src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/PreviewResult.cs

```diff
@@ -13,10 +16,12 @@ public class PreviewResult
 
         public IEnumerable<UpdatePreviewResult> Updated { get; }
 
-        public string Name { get; }
+        public Dictionary<string, SortedSet<string>>? NewSourceMappings { get; }
+
+        public string? Name { get; }
 
         public PreviewResult(
-            string projectName,
+            string? projectName,
```

Comment:
Because one of the supplied values, `UniqueName`, can be null:

https://github.com/NuGet/NuGet.Client/blob/77d0b0f78ae21813fc605fef3e2c62f92ae84a7e/src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/ProjectMetadataContextInfo.cs#L19